features/case_workers/view_claim_evidence.feature

```diff
@@ -0,0 +1,22 @@
+@advocate
```

Comment:
following your suggested dir structure should these advocate features not be in the external_users/advocate directory